oap-server/server-starter/src/main/resources/ui-initialized-templates/elasticsearch/elasticsearch-index.json

```
@@ -0,0 +1,674 @@
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents (All Shards)"
+          }
+        },
+        {
+          "x":8,
+          "y":59,
+          "w":8,
+          "h":11,
+          "i":"8",
+          "type":"Widget",
+          "id":"8",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_docs_total_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents Rate (All Shards)"
+          }
+        },
+        {
+          "x":0,
+          "y":48,
+          "w":8,
+          "h":11,
+          "i":"9",
+          "type":"Widget",
+          "id":"9",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_docs_primary"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents (Primary Shards)"
+          }
+        },
+        {
+          "x":8,
+          "y":48,
+          "w":8,
+          "h":11,
+          "i":"10",
+          "type":"Widget",
+          "id":"10",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_docs_primary_rate"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "widget":{
+            "title":"Documents Rate (Primary Shards)"
+          }
+        },
+        {
+          "x":16,
+          "y":48,
+          "w":8,
+          "h":11,
+          "i":"11",
+          "type":"Widget",
+          "id":"11",
+          "metricTypes":[
+            "readMetricsValues"
+          ],
+          "metrics":[
+            "meter_elasticsearch_index_indices_store_size_bytes_primary"
+          ],
+          "moved":false,
+          "graph":{
+            "type":"Line",
+            "step":false,
+            "smooth":false,
+            "showSymbol":false,
+            "showXAxis":true,
+            "showYAxis":true
+          },
+          "metricConfig":[
+            {
+              "calculation":"byteToMB"
+            }
+          ],
+          "widget":{
+            "title":"Total size of stored index data (Primary Shards) (MB)"
```

```suggestion
            "title":"Total size of indices (Primary Shards) (MB)"
```

I don't think there is an index not stored.